scalastyle-config.xml

```diff
@@ -150,6 +150,13 @@ This file is divided into 3 sections:
       // scalastyle:on println]]></customMessage>
   </check>
 
+  <check customId="visiblefortesting" level="error" class="org.scalastyle.file.RegexChecker" enabled="true">
+    <parameters><parameter name="regex">@VisibleForTesting</parameter></parameters>
+    <customMessage><![CDATA[
+			Are you sure that you want to use @VisibleForTesting ? In most cases, you should use // VisibleForTesting instead.
```

Comment:
It's weird to have `// VisibleForTesting` as a comment. I'd say we just put it in the java doc of the method, e.g.
```
/**
 * This method does something...
 * Exposed for testing.
 */
```
For the custom message here I'd just say the annotation is banned or something.